### PR TITLE
Update Manubot on 2018-01-11

### PR DESCRIPTION
**Pull request by dhimmel**
Manubot now differentiates pubmed and PMC citations. Also author affiliations should be list.

@agitter are there any references where we wanted to cite a pmid but that failed? I don't think the reference from https://github.com/greenelab/deep-review/pull/351/commits/24dfecec1c9c7d447d152020f8b7f9b2e6cdf10e is still cited?

Will update frontmatter in a subsequent PR

### CODE REVIEW

File: build/assets/hypothesis.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<!-- Enable Hypothesis annotation. https://web.hypothes.is/ -->
+<script src="https://hypothes.is/embed.js" async></script>
```
Comment (agitter) [inline build/assets/hypothesis.html]: @cgreene note that this update will enable Hypothesis annotation on the deep review HTML manuscript.

File: content/00.front-matter.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ on {{date}}.
   {%- endif %}<br>
   <small>
   {%- if author.affiliations is defined %}
-     {{author.affiliations}}
+     {{author.affiliations | join('; ')}}
```
Comment (agitter) [inline content/00.front-matter.md]: This looks good for the initial merge.  We can make a shorter version per https://github.com/greenelab/deep-review/issues/700#issuecomment-353462637 in a separate update.

File: content/08.methods.md [hunk 3]
```diff
@@ -9,10 +9,10 @@ We wrote the manuscript in markdown and tracked changes using git.
 Contributions were handled through GitHub, with individuals submitting "pull requests" to suggest additions to the manuscript.
 
 To facilitate citation, we [defined](https://github.com/greenelab/deep-review/blob/master/CONTRIBUTING.md#markdown) a markdown citation syntax.
-We supported citations to the following identifier types (in order of preference): DOIs, PubMed IDs, arXiv IDs, and URLs.
+We supported citations to the following identifier types (in order of preference): DOIs, PubMed Central IDs, PubMed IDs, arXiv IDs, and URLs.
```
Comment (agitter) [inline content/08.methods.md]: Should we remove PubMed IDs now and keep only PubMed Central IDs in this list?

File: content/08.methods.md [hunk 4]
```diff
@@ -9,10 +9,10 @@ We wrote the manuscript in markdown and tracked changes using git.
...
 References were automatically generated from citation metadata by querying APIs to generate [Citation Style Language](http://citationstyles.org/) (CSL) JSON items for each reference.
 [Pandoc](http://pandoc.org/) and [pandoc-citeproc](https://github.com/jgm/pandoc-citeproc) converted the markdown to HTML and PDF, while rendering the formatted citations and references.
-In total, referenced works consisted of {{manuscript_stats.reference_counts.doi}} DOIs, {{manuscript_stats.reference_counts.pmid}} PubMed records, {{manuscript_stats.reference_counts.arxiv}} arXiv manuscripts, and {{manuscript_stats.reference_counts.url}} URLs (webpages as well as manuscripts lacking standardized identifiers).
+In total, referenced works consisted of {{manuscript_stats.reference_counts.doi}} DOIs, {{manuscript_stats.reference_counts.pmcid}} PubMed Central records, {{manuscript_stats.reference_counts.pmid}} PubMed records, {{manuscript_stats.reference_counts.arxiv}} arXiv manuscripts, and {{manuscript_stats.reference_counts.url}} URLs (webpages as well as manuscripts lacking standardized identifiers).
```
Comment (agitter) [inline content/08.methods.md]: Same question here, should we remove PubMed ID counts?  Or are there some that couldn't be converted to PMCIDs?

File: content/metadata.yaml [hunk 5]
```diff
@@ -15,18 +15,21 @@ author_info:
...
+    affiliations:
+      - Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
     funders: GBMF GBMF4552
   - github: brettbj
     name: Brett K. Beaulieu-Jones
     orcid: 0000-0002-6700-1468
     email: brettbe@upenn.edu
-    affiliations: Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
+    affiliations:
+      - Genomics and Computational Biology Graduate Group, Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA
```
Comment (agitter) [inline content/metadata.yaml]: The next author `alxndrkalinin` needs the affiliation as a list.

File: content/new-author-metadata.yaml [hunk 6]
```diff
@@ -1,56 +1,51 @@
...
-    orcid: XXXX-XXXX-XXXX-XXXX
-    email: jane.roe@whatever.edu
-    affiliations: Department of Whatever, University of Something
   - github: luzh2
     name: Zhiyong Lu
     initials: ZL
     orcid: 0000-0001-9998-916X
     email: zhiyong.lu@nih.gov
-    affiliations: National Center for Biotechnology Information and National Library of Medicine, National Institutes of Health, Bethesda, MD
+    affiliations:
```
Comment (agitter) [inline content/new-author-metadata.yaml]: Thanks for catching these updates as well.

File: content/08.methods.md (hunk 3 above)
Comment (dhimmel) [reply content/08.methods.md]: I think I'll have to edit the methods section one final time to make sure things are up to date. Let's see at that time whether we have any PubMed IDs references?

**Comment by agitter** [comment]
> are there any references where we wanted to cite a pmid but that failed

They may have been others, but I don't remember which.  I believe we found workarounds for everything.

**Pull request merged**